dhis-2/dhis-test-performance/src/test/org/hisp/dhis/test/tracker/TrackerTest.java

```diff
@@ -59,64 +61,276 @@ public TrackerTest() {
                 "http://localhost:8080/api/ping") // https://docs.gatling.io/reference/script/http/protocol/#warmup
             .disableCaching(); // to repeat the same request without HTTP cache influence (304)
```

Comment:
Can we default this check for all requests?

         .check(status().is(200))

and only override when you expect different status codes 🤔